Huawei Router Tool/Form1.cs

```diff
@@ -3620,8 +3631,32 @@ private void replyMessageToolStripMenuItem_Click(object sender, EventArgs e)
             rtbSMSContent.Focus();
 
 
-        }
-
+        }
+
+        private void showHidePass_Click(object sender, EventArgs e)
+        {
+            if (textBoxPassword.UseSystemPasswordChar == true)
+            {
+                textBoxPassword.UseSystemPasswordChar = false;
+            }
+            else
+            {
+                textBoxPassword.UseSystemPasswordChar = true;
+            }
+        }
+
+        private void timer1_Tick(object sender, EventArgs e)
+        {
+            if (textBoxPassword.UseSystemPasswordChar == true)
+            {
+                showHidePass.Text = "Show";
+            }
+            else
+            {
+                showHidePass.Text = "Hide";
+            }
+        }

```

Comment:
Show/hide password textbox system (don't forget to add timer to this project if doesn't exist)